app/models/article.rb

```diff
@@ -391,6 +397,10 @@ def video_duration_in_minutes
     hours < 1 ? "#{minutes}:#{seconds}" : "#{hours}:#{minutes}:#{seconds}"
   end
 
+  def video_duration_in_minutes_integer
```

Comment:
I broke this off from the above method so that I could use it [below in the serializer.](https://github.com/thepracticaldev/dev.to/pull/6488/files#diff-89a09f9e11e8944c241f92d9ba6dff4eR12)